### ISSUE TITLE
Prevents opening the global settings dialog on 2017.2

### ISSUE THREAD

**Issue by emilssolmanis** [NixOS/nix-idea#10]
Hey,

After the 2017.2 update this plugin completely kills opening the settings dialog (`Cmd-,` on Mac, `Ctrl-Alt-S` Win). The trace for Mac is

```
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - Either do the specified parameters not match any of the following constructors: [] or the constructors were not accessible for 'class org.nixos.idea.settings.NixIDEASettings' 
org.picocontainer.PicoInitializationException: Either do the specified parameters not match any of the following constructors: [] or the constructors were not accessible for 'class org.nixos.idea.settings.NixIDEASettings'
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getGreediestSatisfiableConstructor(CachingConstructorInjectionComponentAdapter.java:187)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.doGetComponentInstance(CachingConstructorInjectionComponentAdapter.java:92)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.instantiateGuarded(CachingConstructorInjectionComponentAdapter.java:80)
	at com.intellij.util.pico.CachingConstructorInjectionComponentAdapter.getComponentInstance(CachingConstructorInjectionComponentAdapter.java:63)
	at com.intellij.openapi.extensions.AbstractExtensionPointBean.instantiate(AbstractExtensionPointBean.java:75)
	at com.intellij.openapi.options.ConfigurableEP$ClassProducer.createElement(ConfigurableEP.java:341)
	at com.intellij.openapi.options.ConfigurableEP.createConfigurable(ConfigurableEP.java:267)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.a(ConfigurableWrapper.java:58)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:117)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:98)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getId(ConfigurableWrapper.java:194)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.groupConfigurables(ConfigurableExtensionPointUtil.java:269)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.getConfigurableGroup(ConfigurableExtensionPointUtil.java:145)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.getConfigurableGroup(ConfigurableExtensionPointUtil.java:136)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.getConfigurableGroups(ShowSettingsUtilImpl.java:67)
	at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:63)
	at com.intellij.ide.MacOSApplicationProvider$Worker.a(MacOSApplicationProvider.java:118)
	at com.intellij.ide.MacOSApplicationProvider$Worker.b(MacOSApplicationProvider.java:212)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.b(TransactionGuardImpl.java:72)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - IntelliJ IDEA 2017.2  Build #IU-172.3317.76 
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - JDK: 1.8.0_152-release 
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - VM: OpenJDK 64-Bit Server VM 
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - Vendor: JetBrains s.r.o 
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - OS: Mac OS X 
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - Last Action: ShowProjectStructureSettings 
2017-07-18 16:02:41,895 [  94848]  ERROR - openapi.options.ConfigurableEP - Assertion failed: Bundle missed for org.nixos.idea.settings.NixIDEASettings 
java.lang.Throwable
	at com.intellij.openapi.diagnostic.Logger.assertTrue(Logger.java:168)
	at com.intellij.openapi.options.ConfigurableEP.getDisplayName(ConfigurableEP.java:72)
	at com.intellij.openapi.options.ConfigurableEP.toString(ConfigurableEP.java:279)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:119)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:98)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getId(ConfigurableWrapper.java:194)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.groupConfigurables(ConfigurableExtensionPointUtil.java:269)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.getConfigurableGroup(ConfigurableExtensionPointUtil.java:145)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.getConfigurableGroup(ConfigurableExtensionPointUtil.java:136)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.getConfigurableGroups(ShowSettingsUtilImpl.java:67)
	at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:63)
	at com.intellij.ide.MacOSApplicationProvider$Worker.a(MacOSApplicationProvider.java:118)
	at com.intellij.ide.MacOSApplicationProvider$Worker.b(MacOSApplicationProvider.java:212)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.b(TransactionGuardImpl.java:72)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
2017-07-18 16:02:41,896 [  94849]  ERROR - openapi.options.ConfigurableEP - IntelliJ IDEA 2017.2  Build #IU-172.3317.76 
2017-07-18 16:02:41,896 [  94849]  ERROR - openapi.options.ConfigurableEP - JDK: 1.8.0_152-release 
2017-07-18 16:02:41,896 [  94849]  ERROR - openapi.options.ConfigurableEP - VM: OpenJDK 64-Bit Server VM 
2017-07-18 16:02:41,896 [  94849]  ERROR - openapi.options.ConfigurableEP - Vendor: JetBrains s.r.o 
2017-07-18 16:02:41,896 [  94849]  ERROR - openapi.options.ConfigurableEP - OS: Mac OS X 
2017-07-18 16:02:41,896 [  94849]  ERROR - openapi.options.ConfigurableEP - Last Action: ShowProjectStructureSettings 
2017-07-18 16:02:41,896 [  94849]  ERROR - pplication.impl.LaterInvocator - Argument for @NotNull parameter 'pathToBundle' of com/intellij/AbstractBundle.getResourceBundle must not be null 
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'pathToBundle' of com/intellij/AbstractBundle.getResourceBundle must not be null
	at com.intellij.AbstractBundle.getResourceBundle(AbstractBundle.java)
	at com.intellij.openapi.options.ConfigurableEP.getDisplayName(ConfigurableEP.java:73)
	at com.intellij.openapi.options.ConfigurableEP.toString(ConfigurableEP.java:279)
	at java.lang.String.valueOf(String.java:2994)
	at java.lang.StringBuilder.append(StringBuilder.java:131)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getConfigurable(ConfigurableWrapper.java:119)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.cast(ConfigurableWrapper.java:98)
	at com.intellij.openapi.options.ex.ConfigurableWrapper.getId(ConfigurableWrapper.java:194)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.groupConfigurables(ConfigurableExtensionPointUtil.java:269)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.getConfigurableGroup(ConfigurableExtensionPointUtil.java:145)
	at com.intellij.openapi.options.ex.ConfigurableExtensionPointUtil.getConfigurableGroup(ConfigurableExtensionPointUtil.java:136)
	at com.intellij.ide.actions.ShowSettingsUtilImpl.getConfigurableGroups(ShowSettingsUtilImpl.java:67)
	at com.intellij.ide.actions.ShowSettingsAction.perform(ShowSettingsAction.java:63)
	at com.intellij.ide.MacOSApplicationProvider$Worker.a(MacOSApplicationProvider.java:118)
	at com.intellij.ide.MacOSApplicationProvider$Worker.b(MacOSApplicationProvider.java:212)
	at com.intellij.openapi.application.TransactionGuardImpl.a(TransactionGuardImpl.java:86)
	at com.intellij.openapi.application.TransactionGuardImpl.b(TransactionGuardImpl.java:72)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:416)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:399)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:762)
	at java.awt.EventQueue.access$500(EventQueue.java:98)
	at java.awt.EventQueue$3.run(EventQueue.java:715)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:732)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:821)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:649)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:365)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

**Comment by xurei** [comment]
I have the same issue. Disabled the plugin until a fix is published

**Comment by puneetar** [comment]
+1

**Comment by dudinsky** [comment]
This solution was also helpful for me.
+1

**Comment by pSub** [comment]
Should be fixed with release 0.2.1.0, see also #12.
